frontend/src/PsychologistRouter.jsx

```diff
@@ -91,23 +90,22 @@ const PsychologistRouter = () => {
       withNotification
     >
       {modal}
-      <Announcement />
```

Comment:
Why did you remove it here ?

Reply:
I found that it was too messy in the pages with all the callouts and notifications. It also seems unappropriate to put something out from SPE scope inside the pages where user will enter appointments, download bills, etc.